packages/functional-tests/pages/settings/index.ts

```diff
@@ -107,4 +109,22 @@ export class SettingsPage extends SettingsLayout {
     await expect(this.alertBar).toHaveText('Two-step authentication disabled');
     await expect(this.totp.status).toHaveText('Disabled');
   }
+
+  /**
```

Comment:
I've pulled out helper code like this into a separate PR (https://github.com/mozilla/fxa/pull/19426) to prevent this from getting lost if we need to rollback the guard for this route. It should merge cleanly if the other PR lands first.